apps/web/src/services/asset-inventory/collector/collector-main/modules/collector-item-info/CollectorItemStatus.vue

```diff
@@ -30,7 +30,7 @@
                             color="inherit"
                         />
                         <p class="description">
-                            {{ $t('INVENTORY.COLLECTOR.MAIN.SCHEDULED') + $t('INVENTORY.COLLECTOR.MAIN.SCHEDULED_TIME', {hr: state.diffSchedule.diffHour, m: state.diffSchedule.diffMin }) }}
+                            {{ $t('INVENTORY.COLLECTOR.MAIN.SCHEDULED') + " " + $t('INVENTORY.COLLECTOR.MAIN.SCHEDULED_TIME', {hr: state.diffSchedule.diffHour, m: state.diffSchedule.diffMin }) }}
```

Comment:
question: Is this an intended space addition??

Reply:
yup!